plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -684,10 +684,8 @@ end
 
 function ListMenu:_updateItemsBuildUI()
     -- Build our list
-    -- We separate items with a 1px LineWidget (no need for
-    -- scaleBySize, thin is fine)
```

Comment:
Really ?!
I made some esthetic choices (which can be disagreed), added a comment explaining it, and you're making it a 2px width now on most devices,, and removing the comment, as part of this scaleBySize campaign ? :|

Reply:
At 600 DPI it's not a design choice but a bug. I would've changed it to `0.5` but right now this merely functions as a tag for the next step. ;-)

Reply:
More concretely, this'll become something like your proposed ui.elements.commonui.THIN_LINE = scaleBySize(0.5) property.